backend/src/test/kotlin/blue/mild/covid/vaxx/routes/DataCorrectnessRoutesTest.kt

```diff
@@ -79,6 +79,15 @@ class DataCorrectnessRoutesTest : ServerTestBase() {
         // by calling the API and requesting data for patient1.id and expecting Ok status
         // and then comparing correctnessId with the ID you received in the test
         // hint: see first part of the test
+
+        handleRequest( HttpMethod.Get, "${Routes.dataCorrectness}?id=${patient1.id}" ) {
```

Comment:
use some autoformat (Idea has it built in, Ctrl+Alt+L) to remove extra spaces and insert the missing ones